### PR TITLE
clean files

### LINKED ISSUE #231: what are old files
https://github.com/rladies/starter-kit/tree/master/old-files-do-not-delete

### PR DESCRIPTION
**Pull request by maelle**
Fix #231

Now that the R-Ladies guide is at least as complete as the starter kit docs.

CC @katherinesimeon (if you have RoCur links to update elsewhere), @flor14 (if you have links to the mentoring program to update elsewhere)

@yabellini do you think we should delete the two markdown files in Spanish? :thinking: (because they're out of sync) but where do we keep them if they're useful for a translation later.

### CODE REVIEW

File: R/script-to-update-current-chapters.R [hunk 1]
```diff
@@ -1,53 +0,0 @@
-#-----------------------------------------------------------#
-# Goal: Update the "Current-Chapters.csv" programmatically
-# Details about this script:
-# 1) Update the meetup url
```
Comment (maelle) [inline R/script-to-update-current-chapters.R:4]: @ledell @yabellini unless this is useful for something when monitoring chapters? I thought it was an old script no longer used.

File: R/script-to-update-social-media.R [hunk 2]
```diff
@@ -1,50 +0,0 @@
...
-# Goal: Update the social medias from "Current-Chapters.csv" programmatically
-# Details about this script:
-# 1) Update the social media 
-# (Twitter Facebook Instagram Periscope Youtube GitHub Website Slack)
-# function: add_social_media
-
-library(futile.logger)
-library(data.table)
-data <- fread("Current-Chapters.csv")
-
```
Comment (maelle) [inline R/script-to-update-social-media.R:11]: same question

**Comment by maelle** [comment]
Poke @katherinesimeon @yabellini @flor14 @hfrick @ledell It'd be great if you could have a look this week, but if not, no problem, I'll send a new reminder next week. :slightly_smiling_face:

File: R/script-to-update-current-chapters.R (hunk 1 above)
Comment (yabellini) [reply R/script-to-update-current-chapters.R:4]: I never use this script.

File: R/script-to-update-social-media.R (hunk 2 above)
Comment (yabellini) [reply R/script-to-update-social-media.R:11]: The same answer, I never use this.

File: R/script-to-update-current-chapters.R (hunk 1 above)
Comment (ledell) [reply R/script-to-update-current-chapters.R:4]: I am not sure this script saves any time over making an update manually... I don't think it's needed.

File: R/script-to-update-social-media.R (hunk 2 above)
Comment (ledell) [reply R/script-to-update-social-media.R:11]: Same comment as above.  Not needed.

**Comment by yabellini** [comment]
About the Spanish .md, Can we have some 'es' folder or something like that where we can move everything that is in Spanish there so we can check what is useful latter?

**Comment by maelle** [comment]
Good idea @yabellini! Done!

@yabellini @ledell for merging I'll need GitHub reviews (see top right corner).

**Review by yabellini — APPROVED** [review]

**Pull request merged**